docs/apache-airflow-providers-google/operators/cloud/bigquery.rst

```diff
@@ -42,7 +42,7 @@ Create dataset
 To create an empty dataset in a BigQuery database you can use
 :class:`~airflow.providers.google.cloud.operators.bigquery.BigQueryCreateEmptyDatasetOperator`.
 
-.. exampleinclude:: /../../airflow/providers/google/cloud/example_dags/example_bigquery_operations.py
+.. exampleinclude:: /../../tests/system/google/bigquery/test_bigquery_dataset.py
```

Comment:
The build docs chekc failed because file was moved under provider parent directuiey